sdk/examples/upload_disk_snapshots.py

```diff
@@ -186,6 +186,7 @@ def create_disk(image_info, disk_id, sd_name, disks_service):
             format=types.DiskFormat.RAW,
             provisioned_size=provisioned_size,
             initial_size=initial_size,
+            # sparse=False, # In case of iSCSI Storage discomment this.
```

Comment:
sparse=True is valid when using qcow2 format, which is the only format supported
for snapshots.

Only raw=sparse is not supported for block storage domains (FC, iSCSI)